templates/javascript-ethers-hardhat/scripts/whitelist-reader.js

```diff
@@ -5,20 +5,20 @@ const { getServiceData, whitelistBeaconReader } = require("@api3/services");
 async function main() {
   const network = hre.network.name;
 
-  // TODO: Update the mocked contract so that whitelisting is necessary
-  if (
-    network.toLowerCase() === "hardhat" ||
-    network.toLowerCase() === "localhost"
-  )
+  if (network.toLowerCase() === "localhost") {
+    console.log(
+      'Whitelisting is not necessary when using the mocked "RrpBeaconServerMock" contract.'
+    );
     return;
+  }
 
   // Read BeaconReaderExample contract address deployed using deploy.js
   let beaconReaderExampleAddress = null;
   try {
-    // TODO: Simplify
-    ({ beaconReaderExampleAddress } = JSON.parse(
+    const parseResult = JSON.parse(
       fs.readFileSync(`./deployments/${network}.json`).toString()
-    ));
+    );
+    beaconReaderExampleAddress = parseResult.beaconReaderExampleAddress;
```

Comment:
Is this simpler? 😁

Reply:
For the average JS users I'd definitely say so. Combinining the destructuring which binds the value to a let defined variable outside if this try-catch statement is an advanced concept. And I am not even talking about why the whole thing needs to be in braces anyway (I know why, but the reason is pretty stupid)